docs/development/documentation_guidelines.rst

```diff
@@ -150,10 +152,10 @@ called ``securedrop``. This ``securedrop`` directory also contains a
 
      .
      ├── securedrop
-     │   │
-     │  ...
-     │   ├── securedrop
-    ... ...
+     │   │
+     │  ...
+     │   ├── securedrop
+    ... ...
```

Comment:
This looks like an automatic conversion from nonbreaking spaces to regular spaces. Looks content-neutral to me in rendered output.